crates/aptos-telemetry-service/src/prometheus_push_metrics.rs

```diff
@@ -64,66 +65,61 @@ pub async fn handle_metrics_ingest(
             .and_then(|peers| peers.get(&claims.peer_id)),
     );
 
-    let start_timer = Instant::now();
-
-    let filtered_clients = context.metrics_client().iter().filter(|(name, _)| {
-        if claims.chain_id.id() == 3 {
```

Comment:
Can you explain why are these not needed anymore? Also, may be add that context in the PR?

Reply:
Yes, this was an hack I implemented when we had to rerun premainnet to get metrics to proper backends. We dont need these anymore.